_utils/vault.py

```diff
@@ -26,7 +27,7 @@ def _get_token_and_url_from_master():
   if __opts__.get('__role', 'minion') == 'minion':
     private_key = '{0}/minion.pem'.format(pki_dir)
     log.debug('Running on minion, signing token request with key {0}'.format(private_key))
-    signature = salt.crypt.sign_message(private_key, minion_id)
+    signature = base64.b64encode(salt.crypt.sign_message(private_key, minion_id))
```

Comment:
This should be on the corresponding line for encrypting with the master key as well